app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    storage = {}
+
+    def wrapper(*args) -> any:
```

Comment:
```suggestion 
    @wraps(func)
    def wrapper(*args, **kwargs) -> Any:
```
- Good to add an annotation from `functools` module
- Also, it should work with `**kwargs`. Create a unique key using args and kwargs